frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.tsx

```diff
@@ -70,6 +71,13 @@ export function DashboardInfoSidebar({
     query: { model_type: "dashboard", model_id: dashboard.id },
   });
 
+  const isIADashboard = useMemo(
```

Comment:
This name confused me for a couple seconds. Maybe `isInstanceAnalyticsDashboard`? It's a mouthful, or rather an eyeful, but at least it's clear.